rename: fix 'rename' command has two different versions (#3126)
tldr-pages/tldr: pages/linux/rename.md

# rename

> Rename multiple files.
> NOTE: this page refers to the command from the `util-linux` package.
> For the Perl version, see `file-rename` or `perl-rename`.
> Warning: This command has no safeguards and will overwrite files without prompting.

- Rename files using simple substitutions (substitute 'foo' with 'bar' wherever found):

`rename {{foo}} {{bar}} {{*}}`

- Dry-run - display which renames would occur without performing them:

`rename -vn {{foo}} {{bar}} {{*}}`

- Do not overwrite existing files:

`rename -o {{foo}} {{bar}} {{*}}`

- Change file extensions:

`rename {{.ext}} {{.bak}} {{*.ext}}`

- Prepend "foo" to all filenames in the current directory:

`rename {{''}} {{'foo'}} {{*}}`

> - Rename 'foo1', ..., 'foo10', ..., 'foo100' to 'foo001', ..., 'foo010', ..., 'foo100':

Reviewer: ```suggestion
- Rename a group of increasingly numbered files zero-padding the numbers up to 3 digits:
```

# rename

> > Renames multiple files.

Reviewer: ```suggestion
> Rename multiple files.
```
While we're at it, let's also correct this and make it impersonal (really strange that the man page uses third person). Same goes for `rename.ul`.

Author: Fixed, thanks!

> > This is the Perl version of `rename`, which is the default on Debian. See `rename.ul` for the util-linux version.

Reviewer: ```suggestion
> This is the Perl version of `rename`, which is the default on Debian. See `rename.ul` for the util-linux version.
> Warning: This command has no safeguards and will overwrite files without prompting.
```
I'd keep the same warning on both pages.

Author: The Perl version prompts before overwriting. You can force it with `-f` so the warning is inapplicable.

> > This is the perl version of `rename`, which is the default on Debian. See `rename.ul` for the stand-alone version.

Reviewer 2: Maybe make this more explicit? Also capitalize "Perl".

```suggestion
> This is the Perl version of `rename`, which is the default on Debian. See `rename.ul` for the util-linux version.
```

And perhaps to equalize the length of both lines, we could instead consider something like this:

> Rename multiple files using Perl regular expressions.
> For the C-based command included in util-linux, see `rename.ul`.

Author: >Rename multiple files using Perl regular expressions.
For the C-based command included in util-linux, see rename.ul.

This looks a bit strange to me. Changing the first sentence is fine but the `C-based` probably wont mean anything to most people.

Reviewer 2: Yeah, I was unsure as well. In any case, the current version sounds good to me.
